src/activity-logger/background/index.js

```diff
@@ -1,8 +1,17 @@
 import maybeLogPageVisit from './log-page-visit'
+import _ from 'lodash'
```

Comment:
Because of the way lodash organises its functions, if you import the default from `'lodash'` like this, from what I understand, it will pull in the entire lib to the final build unnecessarily. If we do what we've done in the rest of the modules and replace this with `import debounce from 'lodash/debounce'`, it will just cherry-pick that function's needed code and save a lot of space.

Using smarter build tools that support tree shaking can automatically handle it if you import lodash functions as named imports on the standard `'lodash'` package, but I'm pretty sure browserify does not.